U/USalign/build_tarballs.jl

```diff
@@ -0,0 +1,97 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "USalign"
+version = v"2022.9.24"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/pylelab/USalign.git", "944c6a81c453b43d173d8f18ff3197d59650c2cc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd USalign/
+# Remove the -ffast-math flag from the suggested g++ command as required by BinaryBuilder.
+sed -i 's/-ffast-math//g' Makefile
+# The following sed statements are needed to support Linux powerpc64le {libc=glibc}.
+if [[ "${target}" == powerpc64le* ]]; then
+    sed -i 's/\bA0\b/A0_var/g' TMalign.h;
+    sed -i 's/\bB0\b/B0_var/g' TMalign.h;
+    sed -i 's/\bC0\b/C0_var/g' TMalign.h;
+    sed -i 's/\bD0\b/D0_var/g' TMalign.h;
+fi
+# Remove the -static flag to support macOS aarch64 and x86_64.
+if [[ "${target}" == *-apple-darwin* ]]; then
+    sed -i 's/-static//g' Makefile;
+fi
+
+
+if [[ "${target}" == *-mingw* ]]; then
+    # Fix the EXIT_SUCCESS not declared error
+    sed -i '1i#include <cstdlib>' pdbAtomName.cpp
+
+    # Add a dummy pstream.h to allow compilation on Windows:
+
+cat << 'CONTENT' > pstream.h
+#ifndef PSTREAM_H
+#define PSTREAM_H
+#include <iostream>
+namespace redi {
+    class ipstream {
+    public:
+        ipstream() {
+            std::cout << "POSIX is not supported on Linux, so this program cannot deal with gzipped files on this platform.\n";
+        }
+        void open(const std::string&) {
+            std::cout << "ipstream::open is not supported on this platform.\n";
+        }
+        bool good() const { return false; }
+        void close() {}
+        friend ipstream& getline(ipstream& is, std::string& str) { return is; }
+    };
+}
+#endif // PSTREAM_H
+CONTENT
+
+fi
+
+make
+PROGRAM=$(grep "PROGRAM=" Makefile | cut -d '=' -f2)
+for prog in $PROGRAM; do
+    install -Dvm 755 "${prog}" "${bindir}/${prog}"
```

Comment:
Can't check it myself, but for Windows that's probably
```suggestion
    install -Dvm 755 "${prog}${exeext}" "${bindir}/${prog}${exeext}"
```

Reply:
OR maybe
```suggestion
    install -Dvm 755 "${prog}" "${bindir}/${prog}${exeext}"
```
I don't know and can't check what's the actual name of the file on disk.